subsys/bluetooth/CMakeLists.txt

```diff
@@ -14,3 +14,28 @@ add_subdirectory_ifdef(CONFIG_BT_LL_SOFTDEVICE controller)
 add_subdirectory_ifdef(CONFIG_BT_MESH_NRF_MODELS mesh)
 
 add_subdirectory_ifdef(CONFIG_BT_NRF_SERVICES services)
+
+if(CONFIG_BT_EATT OR
+   CONFIG_BT_L2CAP_ECRED OR
+   CONFIG_BT_PER_ADV OR
+   CONFIG_BT_AUDIO)
+  message(WARNING "
+  One or more of the options have been enabled:
+  CONFIG_BT_EATT, CONFIG_BT_L2CAP_ECRED, CONFIG_BT_PER_ADV, CONFIG_BT_AUDIO
+  These bluetooth features have not been qualified, do not use in production."
+  )
+endif()
+
+if(CONFIG_BT_LL_SW_SPLIT)
+  message(WARNING "
+  Software-based BLE Link Layer (BT_LL_SW_SPLIT) option has been enabled.
+  This features has not been qualified, do not use in production."
+  )
+endif()
+
+if(CONFIG_BT_LL_SW_SPLIT)
+  message(WARNING "
+  Bluetooth Mesh support (BT_MESH) option has been enabled.
+  This features has not been qualified, do not use in production."
```

Comment:
```suggestion
  This feature has not been qualified, do not use in production."
```

Reply:
Suggest: "supported for development only."
Instead of "do not use in production"

Reply:
Updated to "supported for development only"